doc/dds-core-xtypes/README.md

```diff
@@ -235,51 +236,53 @@ node.access().member() // member used for access from parent to this node
 ```
 Note: `node.data()` will return a `ReadableDynamicDataRef` or a `WritableDynamicDataRef` depending of the node type.
 
-In order to break the tree iteration, the user call throw a boolean value.
+In order to break the tree iteration, the user can throw a boolean value.
```

Comment:
Good view, my mind was reading can :smile:

Reply:
It took me a bit. :p the meaning was clear, but... :p